addons/web/static/tests/views/fields/boolean_favorite_field.test.js

```diff
@@ -202,3 +202,39 @@ test("FavoriteField in editable list view without label", async () => {
         message: "should not be favorite",
     });
 });
+
+test("FavoriteField in kanban view with readonly view", async () => {
+    await mountView({
+        resModel: "partner",
+        domain: [["id", "=", 1]],
+        type: "kanban",
+        arch: `
+            <kanban>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div>
+                            <field name="bar" widget="boolean_favorite" readonly="1"/>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        `,
+    });
+    expect(`.o_kanban_record .o_field_widget .o_favorite > a i.fa.fa-star`).toHaveCount(1, {
+        message: "should be favorite",
+    });
+    expect(`.o_kanban_record .o_field_widget .o_favorite > a`).toHaveText("Remove from Favorites", {
+        message: `the label should say "Remove from Favorites"`,
```

Comment:
this is not ideal, UX wise... The label could be different when the field is readonly. Or no label at all in this case, and a different cursor.

Reply:
removing the assertion doesn't make it better UX wise...